src/components/Match/Match.css

```diff
@@ -291,3 +291,12 @@
   opacity: .8;
   fill: var(--textColorPrimary);
 }
+
+.chatTeamIcon svg {
+  vertical-align: top;
+  height: 26px !important;
+  width: 26px !important;
+  margin-right: 6px;
```

Comment:
Do we want the teamIcon to be center aligned in the Team column, or left aligned on like the rest of the columns in the table?